tensorflow_examples/models/stack_gan/data_download.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Download CUB data and the Char-CNN-RNN embeddings.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import argparse
+import tensorflow as tf
+assert tf.__version__.startswith('2')
+
+ap = argparse.ArgumentParser()
+ap.add_argument('-dp', '--download_path', required=False, help='Download Path')
+args = vars(ap.parse_args())
+
+data_url = "http://www.vision.caltech.edu/visipedia-data/CUB-200-2011/CUB_200_2011.tgz"
+
+def download_data(download_path):
```

Comment:
If you care about this dataset consider submitting it to tensorflow/datasets.

Reply:
Will do this after uploading the trained weights.